ext/memfrs/memfrs.c

```diff
@@ -517,27 +544,30 @@ void parse_unicode_str(uint8_t* ustr, CPUState *cpu)
     printf("String with size %d/%d\n", length, max_length);
 
     if(length == 0 || length > 256 || max_length ==0 || max_length > 256)
-        return;
+        return NULL;
 
     f_info = memfrs_q_field(justr, "Buffer");
     offset = f_info->offset;
     buf_ptr = *((uint64_t*)(ustr+offset));
-    //cpu_memory_rw_debug( cpu, ustr_ptr+offset, (uint8_t*)&buf_ptr, sizeof(buf_ptr), 0 );
     memfrs_close_field(f_info);
-
-    //printf("Address: %" PRIx64 "\n", buf_ptr);
+    
+    printf("testtesttest\n");
 
     buf = (uint8_t*)malloc(max_length+2);
     str = (char*)malloc(max_length+1);
     memset(str, 0, max_length+1);
+    printf("testtesttest\n");
     cpu_memory_rw_debug( cpu, buf_ptr, buf, max_length, 0 );
+    printf("buf %s\n", buf);
```

Comment:
It seems like a output of the raw buf, followed by a processed unicode output (Filename).
Thereby, is it necessary to keep this ?